packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/components/FieldGuideItemIcon/FieldGuideItemIcon.js

```diff
@@ -1,44 +1,49 @@
 import zooTheme from '@zooniverse/grommet-theme'
 import { Media } from '@zooniverse/react-components'
 import PropTypes from 'prop-types'
+import { Box } from 'grommet'
+
+export default function FieldGuideItemIcon(props) {
+  const { alt, className, height, icon, width } = props
+
+  const placeholder = (
+    <Box
+      background={zooTheme.global.colors['accent-1']}
+      className={className}
+      height={`${height}px`}
+      width={`${width}px`}
+    />
+  )
 
-export default function FieldGuideItemIcon (props) {
-  const { alt, className, height, icon, viewBox, width } = props
   if (icon && Object.keys(icon).length > 0) {
     return (
       <Media
         alt={alt}
         className={className}
         fit='contain'
         height={height}
+        placeholder={placeholder}
         src={icon.src}
         width={width}
-        {...props}
       />
     )
+  } else {
+    return placeholder
   }
-
-  return (
-    <svg className={className} viewBox={viewBox} {...props}>
-      <rect fill={zooTheme.global.colors['accent-1']} height={height} width={width} />
-    </svg>
-  )
```

Comment:
I think returning an svg as a placeholder made this component overcomplicated because it required a viewbox, and then the viewbox would still get applied to a `div` within Media. Please let me know if I missed something by converting the placeholder icon to a colorful Box rather than svg.

Reply:
With changes like this, I usually find myself going back through the git history, to see if there’s an explanation from the original author of this line.

Reply:
Changes seem innocuous to me, though. 🤷🏻‍♂️